### PR TITLE
zh-translation: /docs/examples/platform/endpoints/index.md

### PR DESCRIPTION
**Pull request by ilylia**
ref: https://github.com/servicemesher/istio-official-translation/issues/1468

### CODE REVIEW

File: content/zh/docs/examples/platform/endpoints/index.md [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
 
 {{< text bash >}}
 $ curl --request POST --header "content-type:application/json" --data '{"message":"hello world"}' "http://${EXTERNAL_IP}/echo?key=${ENDPOINTS_KEY}"
 {{< /text >}}
 
-To install Istio for GKE, follow our [Quick Start with Google Kubernetes Engine](/zh/docs/setup/platform-setup/gke).
+按照[使用 Google Kubernetes Engine 快速开始](/zh/docs/setup/platform-setup/gke)的说明为 GKE 安装 Istio。
 
-## HTTP endpoints service
+## HTTP 端点服务{#HTTP-endpoints-service}
```
Comment (yuxiaobo96) [inline content/zh/docs/examples/platform/endpoints/index.md]: 建议 endpoint 保留原文不翻译，以下同理

Comment (ilylia) [reply content/zh/docs/examples/platform/endpoints/index.md]: fixed

**Comment by yuxiaobo96** [comment]
@rootsongjc 
可以 merge

**Pull request merged**